### PR TITLE
[BACK-2679] Resolve dependabot issues

### PR DESCRIPTION
**Pull request by darinkrauss**
- Resolve dependabot issues
- Update to node v20.11.0
- Replace request dependency with axios
- Resolve lint issues
- Add lint via jshint
- Add .nvmrc
- Update .travis.yml
- Bump version
- https://tidepool.atlassian.net/browse/BACK-2679

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v20.11.0
```
Comment (jh-bate) [inline .nvmrc:1]: `v20.10.0` ?

**Review by jh-bate — APPROVED** [review]

File: .nvmrc (hunk 1 above)
Comment (darinkrauss) [reply .nvmrc:1]: Updated PR description.

**Review by jh-bate — APPROVED** [review]
LGTM

**Review by jh-bate — APPROVED** [review]
LGTM

**Pull request merged**